app/models/form_profiles/va5655.rb

```diff
@@ -8,4 +8,13 @@ def metadata
       returnUrl: '/veteran-information'
     }
   end
+
+  private
+
+  def va_file_number_last_four
+    (
+      BGS::PeopleService.new(user).find_person_by_participant_id[:file_nbr].presence ||
+      user.ssn.presence
```

Comment:
What do you think of indenting this line?

Reply:
Can you send me a snippet of how you think it should be indented?

Reply:
I've implemented your suggestion.